resources/snakefiles/mapping.smk

```diff
@@ -120,17 +120,19 @@ rule sort_index_bam:
         aln="output/mapping/{mapper}/mapped_reads/{read_sample}_Mapped_To_{contig_sample}.bam"
     output:
         bam=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.sorted.bam"),
-        bai=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.sorted.bam.bai")
+        index=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.sorted.bam.bai")
     conda:
         "../env/mapping.yaml"
     threads:
         config['threads']['sort_bam']
     benchmark:
-        "output/benchmarks/sort_bam/{mapper}/{read_sample}_Mapped_To_{contig_sample}.sorted.txt"
+        "output/benchmarks/mapping/{mapper}/sort_index_bam/{read_sample}_Mapped_To_{contig_sample}.sorted.txt"
     log:
-        "output/logs/sort_bam/{mapper}/{read_sample}_Mapped_To_{contig_sample}.sorted.log"
+        "output/logs/mapping/{mapper}/sort_index_bam/sort_index_bam/{read_sample}_Mapped_To_{contig_sample}.sorted.log"
     shell:
         """
         samtools sort -o {output.bam} -@ {threads} {input.aln} 2> {log}
         samtools index -b -@ {threads} {output.bam} 2>> {log}
+        touch {output.bam}
```

Comment:
This 'touch' shouldn't be necessary. Is samtools automatically making the filename with 'sorted' appended?

Reply:
I don't think samtools appends anything, that is just added in the rule output. I guess I could probably just omit that to make it more similar to the format of the other rules. 

concoct needs bam indexes, but running `samtools index` doesn't actually have an output, nor does concoct explicit input the indexes, so this got a bit tricky and I started adding `touch` statements everywhere. 

Anyway, I just made commits for both those changes.